test/index.test.js

```diff
@@ -3,9 +3,23 @@
 const chai = require('chai');
 const AWS = require('aws-sdk-mock');
 const aws = require('aws-sdk');
-const certTestData = require('./test-cert-data.json');
+const certIssuedTestData = require('./test-cert-data-issued.json');
+const certAllTestData = require('./test-cert-data-all.json');
+const pendingCertTestData = require('./test-pending-cert-data.json');
 const ServerlessCustomDomain = require('../index.js');
 
+function certTestData(...args) {
+  if (typeof args[0] === 'function') {
+    const cb = args[0];
+    cb(null, certAllTestData);
+    return;
+  }
+
+  const [params, cb] = args;
+  if (params.CertificateStatuses) cb(null, certIssuedTestData);
```

Comment:
Haven't seen this before. What is the first `null` arg here supposed to be for?

Reply:
In node the typical callback is a Error-First Callback, IE it has 2 arguments. By supplying null the code above is saying "Success".

```javascript
fs.readFile(function (err, data) {
  ...
})
```